smart-contract-verifier/smart-contract-verifier-server/src/services/solidity_verifier.rs

```diff
@@ -14,16 +14,31 @@ use crate::{
 };
 use s3::{creds::Credentials, Bucket, Region};
 use smart_contract_verifier::{
-    find_methods, solidity, Compilers, Fetcher, ListFetcher, S3Fetcher, SolcValidator,
-    SolidityClient, SolidityCompiler, VerificationError,
+    find_methods, find_methods_from_compiler_output, solidity, Compilers, Fetcher, ListFetcher,
+    S3Fetcher, SolcValidator, SolidityClient, SolidityCompiler, SoliditySuccess, VerificationError,
 };
 use smart_contract_verifier_proto::blockscout::smart_contract_verifier::v2::{
-    LookupMethodsRequest, LookupMethodsResponse,
+    verify_response::PostActionResponses, LookupMethodsRequest, LookupMethodsResponse,
 };
-use std::{str::FromStr, sync::Arc};
+use std::{collections::HashSet, str::FromStr, sync::Arc};
 use tokio::sync::Semaphore;
 use tonic::{Request, Response, Status};
 
+pub enum VerifyPostAction {
+    LookupMethods,
+}
+
+impl FromStr for VerifyPostAction {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "lookup-methods" => Ok(VerifyPostAction::LookupMethods),
```

Comment:
please add this option to comment on this protobuf field. It will make this field appear in comment in swagger

Reply:
btw, why kebab-case?

Reply:
https://github.com/blockscout/blockscout-rs/blob/fe12b77acfceb8d81f5d056e1d6d7aafdd656e43/smart-contract-verifier/smart-contract-verifier-proto/proto/v2/smart-contract-verifier.proto#L135-L137
It was added in the previous PR

Reply:
@sevenzing What is your suggestion instead of kebab-case?